src/libraries/System.Net.Quic/tests/FunctionalTests/MsQuicTests.cs

```diff
@@ -17,7 +18,6 @@ namespace System.Net.Quic.Tests
     [ConditionalClass(typeof(QuicTestBase<MsQuicProviderFactory>), nameof(IsSupported))]
     public class MsQuicTests : QuicTestBase<MsQuicProviderFactory>
     {
-        readonly ITestOutputHelper _output;
         private static ReadOnlyMemory<byte> s_data = Encoding.UTF8.GetBytes("Hello world!");
 
         public MsQuicTests(ITestOutputHelper output)
```

Comment:
If you moved `_output` to the base, I would also move the initialization to the base ctor, i.e.:

```suggestion
        public MsQuicTests(ITestOutputHelper output)
            : base(output)
        { }
```

Reply:
+1

I really with XUnit would just let us write test output without having to pass around this silly ITestOutputHelper thing, but if it's possible, I don't know how to do it.

Reply:
we can write out but that can just mingle as plain text. ITestOutputHelper is instantiated for each test, it shows up only on failures and it is stored for each test separately in testResult.xml so it can be parsed out and stored in database (not sure if we actually do that but we used to preserve the result file)

Reply:
I know. I just wish XUnit would let me do `TestOutput.WriteLine(...)` or something like that, instead of having to explicitly pass around the ITestOutputHelper thing.

Reply:
That would be easy enough if we rename `_output`  to `TestOutput`. :) We just need to drag the instance through constructors but that does not impact the use.